CMakeLists.txt

```diff
@@ -22,9 +22,18 @@ cmake_minimum_required(VERSION 3.0)
 project(MIVisionX)
 set(VERSION "2.2.0")
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+if(UNIX)
+  include(GNUInstallDirs CACHE)
+else()
+  set(CMAKE_INSTALL_LIBDIR "lib")
+  set(CMAKE_INSTALL_DATADIR "share")
+  set(CMAKE_INSTALL_INCLUDEDIR "include")
+  set(CMAKE_INSTALL_BINDIR "bin")
+endif()
```

Comment:
I'm not sure I follow why there's this if/else. Those are the directory names that GNUInstallDirs would be using on Windows anyway. On Linux, without setting a default for the `CMAKE_INSTALL_LIBDIR`, you're going to end up with your libraries in `/opt/rocm/lib64` on RHEL and SLES, which I don't think is the intent.

I think @arvindcheru had it right. This whole if/else should just be:
```cmake
set(CMAKE_INSTALL_LIBDIR "lib" CACHE STRING "Library install directory")
include(GNUInstallDirs)
```